src/screens/TokenSwap.js

```diff
@@ -289,24 +316,32 @@ const TokenSwap = () => {
                 </View>
               </View>
 
-              { showQuote && (
+              { showQuote && quote && (
```

Comment:
There is a split second screen change when cleaning up the state where `quote` is `null` but `showQuote` is still `true`.
This makes an error since we use properties of the quote in this block.